src/pages/DashboardPage.tsx

```diff
@@ -26,6 +26,17 @@ function DashboardPage(props: Props) {
     setTitle(t('dashboard.title'));
   }, []);
 
+  var current = new Date();
+  let hour = current.getHours();
+  const BIERTIJD = 16;
+  var joke = false;
+  var isBiertijd = ((~~hour) >== BIERTIJD);
```

Comment:
`>==` is niets, `>=` wel. Bij equality checking moet je wél `===` doen.

Reply:
Verder mogen de uiterste haakjes `()` weg.

Reply:
Oh, en in plaats van `~~hour` kun je het beste `Math.floor(hour)` doen.

Reply:
https://infoscherm.gewis.nl/backoffice/api.php?query=alcoholtijd